docs/rhoas/rhoas-produce-consume/quickstart.yml

```diff
@@ -0,0 +1,25 @@
+apiVersion: console.openshift.io/v1
+kind: QuickStarts
+metadata:
+  name: Produce and consume in Rhoas Cli
+  annotations:
+    draft: false
```

Comment:
I would suggest changing this to `draft: true`. That way we can go ahead and merge this PR, and handle the content refinement in a follow-up PR.